README.md

```diff
@@ -82,9 +82,12 @@ $ cross rustc --target powerpc-unknown-linux-gnu --release -- -C lto
 
 ## Configuration
 
-You can place a `Cross.toml` file in the root of your Cargo project or use a
-`CROSS_CONFIG` environment variable to tweak `cross`'s behavior. The format
-of `Cross.toml` is documented in [docs/cross_toml.md](docs/cross_toml.md).
+You have three options to configure `cross`:
+1. Write the configuration directly in your `Cargo.toml`, under `[package.metadata.cross]`
+2. Place a `Cross.toml` file in the root of your Cargo project
+3. Set the `CROSS_CONFIG` environment variable, in order to tell `cross` where it should search for the config
```

Comment:
it might be confusing with how you'll need to use package.metadata all the time in Cargo.toml, maybe that should be explained or demonstrated

Reply:
We could even promote its use so we can deprecate the old file and eventually remove it.

Reply:
I added some more documentation for this, including an example for the first configuration option.